core/templates/components/copy-url/copy-url.component.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the copy message tooltip.
+ */
+
+import { Component, Input } from '@angular/core';
+import { Clipboard } from '@angular/cdk/clipboard';
+import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
+
+@Component({
+  selector: 'copy-url',
```

Comment:
Please use Oppia prefix with all selectors.

```suggestion
  selector: 'oppia-copy-url',
```

Reply:
done